joblib/externals/loky/__init__.py

```diff
@@ -22,4 +22,4 @@
            "wrap_non_picklable_objects", "set_loky_pickler"]
 
 
-__version__ = '2.7.0'
+__version__ = '3.0.0dev0'
```

Comment:
Could you please sync with loky 2.8.0 ? I think only this line will change.

Reply:
yes.